lib/elixir/lib/string.ex

```diff
@@ -2383,7 +2412,7 @@ defmodule String do
       false
 
   """
-  @spec contains?(t, t | [t] | :binary.cp()) :: boolean
+  @spec contains?(t, pattern | [<<>>]) :: boolean
```

Comment:
```suggestion
  @spec contains?(t, [t] | pattern) :: boolean
```

It can be a list of any strings, including empty ones.

Reply:
I was wrong here, because ` [<<>>]` means a list of empty strings or an empty list.